cmd/qgb/relayer/cmd.go

```diff
@@ -127,14 +133,79 @@ func Command() *cobra.Command {
 			// creating the p2p querier
 			p2pQuerier := p2p.NewQuerier(dht, logger)
 			retrier := helpers.NewRetrier(logger, 5, 15*time.Second)
+
+			// checking if the provided home is already initiated
+			isInit := store.IsInit(logger, config.Home, store.InitOptions{NeedEVMKeyStore: true})
+			if !isInit {
+				logger.Info("please initialize the EVM keystore using the `relayer keys add/import` command")
+				return store.ErrNotInited
+			}
+
+			// creating the data store
+			openOptions := store.OpenOptions{HasEVMKeyStore: true}
+			s, err := store.OpenStore(logger, config.Home, openOptions)
+			if err != nil {
+				return err
+			}
+			defer func(s *store.Store, log tmlog.Logger) {
+				err := s.Close(log, openOptions)
+				if err != nil {
+					logger.Error(err.Error())
+				}
+			}(s, logger)
+
+			if !common.IsHexAddress(config.evmAccAddress) {
+				logger.Error("provided address is not a correct EVM address", "address", config.evmAccAddress)
+				return nil // should we return errors in these cases?
+			}
+
+			addr := common.HexToAddress(config.evmAccAddress)
+			if !s.EVMKeyStore.HasAddress(addr) {
+				logger.Info("account not found in keystore", "address", config.evmAccAddress)
+				return nil
+			}
+
+			logger.Info("loading EVM account", "address", addr.String())
+
+			var acc accounts.Account
+			for _, storeAcc := range s.EVMKeyStore.Accounts() {
+				if storeAcc.Address.String() == addr.String() {
+					acc = storeAcc
+				}
+			}
+
+			passphrase := config.Passphrase
+			// if the passphrase is not specified as a flag, ask for it.
+			if passphrase == "" {
+				logger.Info("please provide the account passphrase")
+				bzPassphrase, err := term.ReadPassword(int(os.Stdin.Fd()))
+				if err != nil {
+					return err
+				}
+				passphrase = string(bzPassphrase)
+			}
+
+			err = s.EVMKeyStore.Unlock(acc, passphrase)
+			if err != nil {
+				logger.Error("unable to load the EVM private key")
+				return err
+			}
+			defer func(EVMKeyStore *keystore.KeyStore, addr common.Address) {
```

Comment:
related to the above comment, could we deduplicate some code here for the relayer and the orchestrator?

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/277